oss-internship-2020/sapi_lodepng/examples/main_unit_test.cc

```diff
@@ -0,0 +1,225 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <filesystem>
+
+#include "gtest/gtest.h"
+#include "sandbox.h"
+#include "sandboxed_api/util/status_matchers.h"
+
+using sapi::IsOk;
+using testing::Eq;
+using testing::NotNull;
+
+namespace {
+
+// use the current path
+const std::string images_path = std::filesystem::current_path().string();
```

Comment:
Would it be ok to create a temporary directory (using [mkdtemp](https://man7.org/linux/man-pages/man3/mkdtemp.3.html)) in `/tmp` and then delete the files created and the directory? In this case, there isn't any reason to keep the `images_path` flag in the `main_sandboxed` binary since everything is removed, right?